src/sentry/south_migrations/0499_auto__add_field_organizationmember_inviter__add_field_organizationmemb.py

```diff
@@ -0,0 +1,1479 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding field 'OrganizationMember.inviter'
+        db.add_column('sentry_organizationmember', 'inviter',
+                      self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(blank=True, related_name=u'sentry_inviter_set', null=True, to=orm['sentry.User']),
+                      keep_default=False)
+
+        # Adding field 'OrganizationMember.invite_status'
+        db.add_column('sentry_organizationmember', 'invite_status',
+                      self.gf('django.db.models.fields.PositiveSmallIntegerField')(default=0),
```

Comment:
Should this field start off as a nullable field? Adding a non-nullable field locks the table in postgres from what I remember.

Reply:
Ah okay, this can be nullable